app/lib/pre_assembly/batch.rb

```diff
@@ -159,10 +158,7 @@ def incomplete_status
       { status: 'error', message: 'pre_assemble did not complete' }
     end
 
-    # Discover object containers from a manifest.
-    # The relative path to the container is supplied in one of the
-    # manifest columns. The column name to use is configured by the
-    # user invoking the pre-assembly script.
+    # Discover object containers from the manifest file suppled in the bundle_dir.
```

Comment:
is this always `manifest.csv` ?   we have a method for it that seems to always return the hardcoded string.  Which may be ok ...but maybe having this comment say `manifest.csv` file would be useful to distinguish it from "file_manifest" ?   Not sure.

Reply:
I'll have it say "object_manifest" ... give we hardcode the filename elsewhere as you note, I'm reluctant to hardcode it again in a comment.